Block/Checkout/Onepage/Success/OfflineAdditionalInformation.php

```diff
@@ -29,14 +37,29 @@ public function __construct(
      */
     protected function _toHtml()
     {
-        $paymentData = $this->_checkoutSession->getLastRealOrder()->getPayment()->getData();
```

Comment:
It might probably easier and shorter to go with
```php
$order       = $this->_checkoutSession->getLastRealOrder();
$paymentData = $order->getPayment()->getData();
$paymentType = isset($paymentData['additional_information']['payment_type']) ? $paymentData['additional_information']['payment_type'] : null;
```

and in the IF-conditional block can also use 
```php
$orderCurrency = $order->getOrderCurrency()->getCurrencyCode();
```

Reply:
Done